### ISSUE TITLE
Request: Add "heavy hash" algorithm for oBTC (optical bitcoin)

### ISSUE THREAD

**Issue by Feythelus** [trexminer/T-Rex#860]
Hi,

Could I please request that the "heavy hash" algorithm is added for support of mining oBTC (optical bitcoin). 

This is a project that has some outstanding potential having created hardware that has an ultra light energy footprint as something of an "energy efficient" bitcoin. There is a lot of mining at the moment with various miners, however the reliability and stability of T-Rex would greatly improve market adoption and interest of the project.

https://www.powx.org/opow

Thank you,

-F

**Comment by sonicdan** [comment]
Yes, team please include algo for oBTC (heavy hash) with LHR unlock by default would be highly appreciated! Thanks in advance!

**Comment by derp55555** [comment]
+1
